XCUITests/TranslationSnackBarTest.swift

```diff
@@ -5,15 +5,46 @@
 import XCTest
 
 class TranslationSnackBarTest: BaseTestCase {
-    func testSnackBar() {
+    
+    // // This test checks to the correct functionalty of the Translation prompt and Translation is done corrrectly using Google
+    func testSnackBarDisplayed() {
         userState.localeIsExpectedDifferent = true
         navigator.openURL(path(forTestPage: "manifesto-zh-CN.html"))
         waitUntilPageLoad()
         navigator.goto(TranslatePageMenu)
         waitForExistence(app.buttons["TranslationPrompt.doTranslate"])
+        app.buttons["TranslationPrompt.doTranslate"].tap()
```

Comment:
Isn't this action do this tap? `https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L898` 
Whenever it is possible using the FxScreenGraph is better. If some actions in this menu are missing this PR is a good moment to add them :)